haiku-apps/eventual/eventual-1.0.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="A time management system"
+DESCRIPTION="Eventual is a personal time management system for Haiku. \
+Intended to be a full-featured organizer. "
+HOMEPAGE="https://sourceforge.net/projects/eventual/"
+COPYRIGHT="2011 Alexey Burshtein
+	2001-2002 Kevin H. Patterson"
+LICENSE="MIT
+	BSD (3-clause)"
+REVISION="1"
+srcGitRev="2fd70537847d3b9712e96ebbc4ff37d5e961b480"
+SOURCE_URI="https://github.com/HaikuArchives/Eventual/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="229cd7fa206da0606d32fc0db1f44a5eee20f0e8428e3dec141e4aab272ead68"
+SOURCE_FILENAME="Eventual-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="Eventual-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 ?x86_64"
+
+PROVIDES="
+	eventual = $portVersion
+	app:Eventual = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
+BUILD()
+{
+	cd src/Libraries
+	mkdir -p ../lib
+	for LIBDIR in */ ; do
+		LIBDIR=${LIBDIR%?}
+		NAME=lib$LIBDIR.so
+		make OBJ_DIR=objects -C $LIBDIR
+		LIB=$LIBDIR/objects/$NAME
+		cp $LIB ../EventEditor/$NAME
+		cp $LIB ../EventServer/$NAME
+		cp $LIB ../PreferencesPreflet/$NAME
+		cp $LIB ../lib/$NAME
+	done
+	cd ../
```

Comment:
```diff
 BUILD()
 {
-	cd src/Libraries
-	mkdir -p ../lib
-	for LIBDIR in */ ; do
-		LIBDIR=${LIBDIR%?}
-		NAME=lib$LIBDIR.so
-		make OBJ_DIR=objects -C $LIBDIR
-		LIB=$LIBDIR/objects/$NAME
-		cp $LIB ../EventEditor/$NAME
-		cp $LIB ../EventServer/$NAME
-		cp $LIB ../PreferencesPreflet/$NAME
-		cp $LIB ../lib/$NAME
-	done
-	cd ../
+	cd src
+	mkdir -p lib
+	for LIBDIR in Libraries/* ; do
+		make OBJ_DIR=objects -C $LIBDIR
+		NAME=lib${LIBDIR#Libraries/}.so
+		mv $LIBDIR/objects/$NAME lib
+		ln -s ../lib/$NAME EventEditor
+		ln -s ../lib/$NAME EventServer
+		ln -s ../lib/$NAME PreferencesPreflet
+	done
 	make OBJ_DIR=objects -C EventEditor
 	make OBJ_DIR=objects -C EventServer
 	make OBJ_DIR=objects -C PreferencesPreflet
 }
```
@bach5000, here's a variant which produces the exact same package as your current `BUILD()`.
There are 2 small advantages: it's 3 lines smaller, and it does not make 4 copies of each lib. It just moves it to src/lib/ and creates symlinks so that each app can be built.
What do you think?

(Sorry, BTW, for this late comment.)

Reply:
@fbrosson,
It would be better to not make 4 copies of each lib, but I only did it because the build instructions (https://github.com/HaikuArchives/Eventual/blob/master/INSTALL) said so. If there are no conflicts building then I suppose we should go ahead and use it 😃 

Also, late comments are fine. I've made plenty late comments multiple times before 😬